network/topics/scoring.go

```diff
@@ -249,3 +253,13 @@ func filterCommitteesForTopic(topic string, committees []*storage.Committee) []*
 	}
 	return topicCommittees
 }
+
+// truncateStats returns the subnets in a small format topicNum=ti,fmd,mmd,imd
+func truncateStats(filtered map[string]*pubsub.TopicScoreSnapshot) []string {
+	filteredStrings := make([]string, 0, len(filtered))
+	for topic, snapshot := range filtered {
+		formatted := fmt.Sprintf("%s=%d;%0.3f;%0.3f;%0.3f", topic, snapshot.TimeInMesh, snapshot.FirstMessageDeliveries, snapshot.MeshMessageDeliveries, snapshot.InvalidMessageDeliveries)
+		filteredStrings = append(filteredStrings, formatted)
+	}
+	return filteredStrings
+}
```

Comment:
I think @moshe-blox proposed an even further reduction by setting it all in a single string, i.e. applying a join with " " for the list of strings that this function returns. Is that right @moshe-blox?

Reply:
@MatheusFranco99 yeah i agree: https://github.com/ssvlabs/ssv/pull/1699#discussion_r1745627722